Basic-Car-Maintenance/Shared/Dashboard/Views/DashboardView.swift

```diff
@@ -30,20 +30,24 @@ struct DashboardView: View {
         NavigationStack {
             List {
                 ForEach(viewModel.searchedEvents) { event in
-                    VStack(alignment: .leading, spacing: 8) {
+                    VStack(alignment: .leading) {
```

Comment:
```suggestion
                    VStack(alignment: .leading, spacing: 4) {
```
changing the spacing to 4, just cause personal preference when I was looking at it in the simulator